### PR TITLE
fix: added dash when an invoice is not using our format

### PR DESCRIPTION
**Pull request by sstefdev**
- added dash where we show data from invoice (this will only show if the invoice is not following our invoice format)

## Summary by CodeRabbit

- **Bug Fixes**
  - Enhanced robustness of the invoice display by adding default values or placeholders for missing data. This update prevents rendering errors and ensures smooth display even when certain fields are undefined.

### CODE REVIEW

File: packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte [hunk 1]
```diff
@@ -312,16 +314,16 @@
...
+                      {item.name || "-"}
                     </p>
                   </th>
-                  <td>{item.quantity}</td>
+                  <td>{item.quantity || "-"}</td>
                   <td
                     >{formatUnits(item.unitPrice, currency?.decimals ?? 18)}</td
                   >
                   <td>{formatUnits(item.discount, currency?.decimals ?? 18)}</td
                   >
```
Comment (MantisClone) [inline packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte:325]: Do you need `|| "-"` on `item.unitPrice` and `item.discount` too?

Also, what's up with the formatting on these two lines?

File: packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte [hunk 2]
```diff
@@ -272,12 +274,12 @@
           {#each firstItems as item, index (index)}
             <tr class="table-row item-row">
               <th scope="row" class="item-description">
-                <p class="truncate description-text">{item.name}</p>
+                <p class="truncate description-text">{item.name || "-"}</p>
               </th>
-              <td>{item.quantity}</td>
+              <td>{item.quantity || "-"}</td>
               <td>{formatUnits(item.unitPrice, currency?.decimals ?? 18)}</td>
               <td>{formatUnits(item.discount, currency?.decimals ?? 18)}</td>
```
Comment (MantisClone) [inline packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte:281]: Do you need `|| "-"` here too?

**Review by MantisClone — APPROVED** [review]
Looks good! Please resolve the questions and merge when ready. :+1:

File: packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte [hunk 3]
```diff
@@ -279,12 +281,12 @@
...
-              <td>{Number(item.tax.amount)}</td>
+              <td>{Number(item.tax.amount || "-")}</td>
```
Comment (aimensahnoun) [inline packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte:289]: We should also make sure to add here and in the discount and such, these might now be available in some requests.

**Review by aimensahnoun — CHANGES_REQUESTED** [review]

**Review by aimensahnoun — APPROVED** [review]

**Pull request merged**